### PR TITLE
[FABCJ-100] Java Contract Quick Start

### PR DESCRIPTION
**Pull request by mbwhite**
Signed-off-by: Matthew B White <whitemat@uk.ibm.com>

#### Type of change

- Documentation update

#### Description

Taking a quick start tutorial originally written as a github gist for RocketChat users.
Should be pushed into the main documentation

### CODE REVIEW

**Review by mastersingh24 — COMMENTED** [review]
more to come after I test this

**Review by pamandrejko — CHANGES_REQUESTED** [review]
@mbwhite 

To get started, I addressed Gari's comments and made some edits and formatting changes to align with our other tutorials.

I'd still like to address the following items:
- [ ] Need to standardize on the use of the terms `smart contract` and `chaincode` throughout the text.
- [ ] I think we should refactor these instructions to use the new Test network instead of basic-network
- [ ] I think we need to refactor these instructions to use the new cc lifecyle.
- [ ] We need to identify the appropriate location in the toc for this content - and align with the rest of our tutorial strategy.
- [ ] Don't use the fabric tools CLI - instead use the commands as Nik does in the test network.
- [ ] Decide on an appropriate title.

I'll continue to work on these changes.

File: docs/README.md [hunk 1]
```diff
@@ -69,7 +69,7 @@ directory. Note: you may need to adjust depending on your OS.
 
 Prereqs:
  - [Python 3.7](https://wiki.python.org/moin/BeginnersGuide/Download)
- - [Pipenv](https://docs.pipenv.org/en/latest/#install-pipenv-today)
+ - [Pipenv](https://pipenv.kennethreitz.org/en/latest/#install-pipenv-today)
```
[the remark this answers is not in the document]
Comment (mastersingh24) [reply docs/README.md:72]: he's the guy who actually wrote it ... not sure why he had to move it .. maybe he stopped paying for the old domain name and wanted to advertise his own name  :-)

File: docs/source/tutorial/javacontract_quickstart.md [hunk 2]
```diff
@@ -0,0 +1,287 @@
...
+* (Optional) [Setup Logspout](#setup-logspout).
+* [Install and instantiate the Fabcar chaincode](#install-and-instantiate-the-fabcar-chaincode).
+* [Execute transaction functions](#execute-a-transaction-functions).
+* [Update the ledger](#update-the-ledger).
+* [Debugging a smart contract](#debugging-a-smart-contract).
+
+## Prerequisites
+
+Before you begin, you need to install some prerequisite technology required by the tutorial.
+
```
Comment (joealewine) [inline docs/source/tutorial/javacontract_quickstart.md:19]: install:

Don't need "some prerequisite..."

File: docs/source/tutorial/javacontract_quickstart.md [hunk 3]
```diff
@@ -0,0 +1,287 @@
...
+ * [**git**](https://git-scm.com/downloads) Download the latest version of git if you do not already have it installed.
+ * [**Docker and Docker Compose**]((../prereqs.html)) Docker help developers and administrators create standard environments for building and running applications and smart contracts.
+
+**Note:** You do not need to have Java, Node or the Fabric `peer` commands installed, but for development you will at some point need these tools.
+
```
Comment (joealewine) [inline docs/source/tutorial/javacontract_quickstart.md:24]: Development of what?

File: docs/source/tutorial/javacontract_quickstart.md [hunk 4]
```diff
@@ -0,0 +1,287 @@
...
+[GitHub](https://www.github.com) repository called `fabric-samples`. As you're going to run the tutorial on your machine, your first task is to download the
+`fabric-samples` repository.  
+
+* Create a local 'working directory' to clone the example repository.
+* Follow the instructions to [Install Samples, Binaries, and Docker images](../install.html).
+
+### Start a Fabric network
+
+Before you can run and test the smart contract, you will need a Fabric network.  Run the following commands to start the Fabric Network using the `basic-network` sample. These commands deploy and start the peer, orderer, CA, and CouchDB containers that are required in order to install and run the smart contract. It also creates a channel named `mychannel` that is used later in the tutorial.
+
```
Comment (joealewine) [inline docs/source/tutorial/javacontract_quickstart.md:41]: Fabric network, not Fabric Network

**Comment by mbwhite** [comment]
Closing to await https://jira.hyperledger.org/browse/FAB-17269

**Pull request closed without merging**